packages/panna-sdk/src/react/components/buy/select-buy-provider-step.tsx

```diff
@@ -39,6 +43,10 @@ export function SelectBuyProviderStep({ form }: SelectBuyProviderStepProps) {
   // Get available providers for the country
   const availableProviders = useMemo(() => {
     if (!country?.code) return [];
+    const supportedCountries = new Set(
```

Comment:
The variable `supportedCountries` is actually a boolean (result of `.has()`), not a collection. This is confusing and could lead to bugs during future maintenance. Should be named `isCountrySupported` or similar?